proof/invariant-abstract/ARM/ArchEmptyFail_AI.thy

```diff
@@ -52,18 +52,20 @@ lemma decode_tcb_invocation_empty_fail[wp]:
   "empty_fail (decode_tcb_invocation a b (ThreadCap p) d e)"
   by (simp add: decode_tcb_invocation_def split: invocation_label.splits | wp | wpc | intro conjI impI)+
 
-crunch find_pd_for_asid, get_master_pde, check_vp_alignment,
-                   create_mapping_entries, ensure_safe_mapping, get_asid_pool, resolve_vaddr
+crunch
+  find_pd_for_asid, get_master_pde, check_vp_alignment, create_mapping_entries, ensure_safe_mapping,
+  get_asid_pool, resolve_vaddr, decode_sgi_signal_invocation
   for (empty_fail) empty_fail[wp]
   (simp: kernel_object.splits arch_kernel_obj.splits option.splits pde.splits pte.splits)
 
+declare decode_sgi_signal_invocation_empty_fail[simp]
```

Comment:
I'm never sure about this, but we sometimes declare empty_fails with `intro!` etc, do we want to be consistent with these?